tests/phpunit/tests/option/wpAutoloadValuesToAutoload.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Tests for the wp_autoload_values_to_autoload function.
+ *
+ * @group Option
+ *
+ * @covers ::wp_autoload_values_to_autoload
+ */
+class Tests_Option_wpAutoloadValuesToAutoload extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 42441
+	 */
+	public function test_wp_autoload_values_to_autoload() {
+		$this->assertSameSets( array( 'yes', 'on', 'auto-on', 'auto' ), wp_autoload_values_to_autoload() );
+	}
```

Comment:
Let's add a test for the filter here, to cover that it is possible to remove autoload values from the array, but not to add any.

Reply:
done